backend/src/data/repositories/interview-note/interview-note.repository.ts

```diff
@@ -22,6 +22,7 @@ class InterviewNote {
       .select()
       .where({ interviewId })
       .withGraphJoined('author(withoutPassword).[userDetails]')
+      .orderBy('createdAt', 'desc')
```

Comment:
```suggestion
      .orderBy('createdAt', SortOrder.DESC)
```
create an enum for it in the shared module